requirements/edx/base.txt

```diff
@@ -38,6 +37,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.2.1#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 -e common/lib/xmodule
+geoip2==2.9.0
```

Comment:
Is this updated in the right place?  I think we moved the requirements to the base.in file.  I also believe this file is auto generated from the base.in file.


Did I miss the update to the .in file?

See comments at the top of base.txt.

Reply:
Looks like this is the case for all the requirements file changes in this PR.